tests/format/flow/literal/test.js

```diff
@@ -0,0 +1,11 @@
+const fooBazBar1:'Multiline string\
```

Comment:
Please rename the file for the name to include "multiline" and use the snippet from this comment: https://github.com/prettier/prettier/pull/13274#issuecomment-1222488611

Same for TS.